snuba/cli/devserver.py

```diff
@@ -121,6 +121,23 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 "--max-query-workers=1",
             ],
         ),
+        (
+            "subscriptions-consumer-sessions",
```

Comment:
Since this is still very early/experimental I don't think we should include in the devserver and have everyone who runs Sentry in development mode run it yet. What do you think about omitting it from here and just running the subscriptions consumer manually for the time being if needed?

Reply:
I changed the base branch here to a feature branch I am working on.. I reckon should be okay until we decide to get this into production, yeah?